pkgs/tools/backup/borgmatic/default.nix

```
@@ -9,18 +9,19 @@
 , installShellFiles
 , borgmatic
 , testers
+, withApprise ? false
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "borgmatic";
-  version = "1.8.1";
+  version = "1.8.4";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-XbihTQJtoiRRfwjMCP+XEPmbt7//zFPx1fIWOvn92Nc=";
+    sha256 = "sha256-ZiEFiVP6C0LqXQy3EF+obLhxorvcpOCo2c9E+W9YnHs=";
   };
 
-  nativeCheckInputs = with python3Packages; [ flexmock pytestCheckHook pytest-cov ];
+  nativeCheckInputs = with python3Packages; [ apprise flexmock pytestCheckHook pytest-cov ];
```

This is more declarative, but feel free to ignore this if you disagree.

```suggestion
  nativeCheckInputs = with python3Packages; [ flexmock pytestCheckHook pytest-cov ] ++ passthru.optional-dependencies.apprise;
```